.github/workflows/deploy.yml

```
@@ -0,0 +1,106 @@
+          version: 10
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Build project
+        run: pnpm build
+        env:
+          DB_FILE_NAME: ${{ secrets.DB_FILE_NAME }}
+          SECRET_API_KEY: ${{ secrets.SECRET_API_KEY }}
+
+deploy:
```

actionlint (1.7.7)

unexpected key "deploy" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)